src/onnx/onnx.cpp

```diff
@@ -701,14 +700,18 @@ struct onnx_parser
         if(contains(info.attributes, "auto_pad"))
         {
             auto weight_lens = weights->get_shape().lens();
-
             std::vector<std::size_t> k_lens(weight_lens.begin() + 2, weight_lens.end());
             cal_auto_padding_size(info,
                                   values,
                                   k_lens,
                                   values["dilation"].to_vector<std::size_t>(),
                                   in_lens,
                                   padding);
+            auto auto_pad = info.attributes["auto_pad"].s();
+            if(auto_pad.find("SAME") != std::string::npos)
+            {
+                values["padding_mode"] = to_value(op::padding_mode_t::same);
```

Comment:
If you removed the `padding_mode` from pooling then this can be removed.

Reply:
I removed the padding_mode from pooling. Here is for convolution.